docs/develop/smart-contracts/outposts/osmosis.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 1
+---
+
+# Osmosis Outpost
+
+[The Osmosis Outpost](https://app.evmos.org/dapps/defi/osmosis) is an Outpost that leverages
+the [IBC Transfer Extension](https://docs.evmos.org/develop/smart-contracts/evm-extensions/ibc-transfer)
+to interact with the Osmosis [IBC-hooks middleware](https://github.com/osmosis-labs/osmosis/tree/main/x/ibc-hooks) 
+and the Osmosis SwapRouter and Cross Chain Swaps contract.
+It provides an easy-to-use interface for executing token swaps between Osmosis and Evmos 
+and routing them back to the Evmos chain. 
+
+## Key Components
```

Comment:
markdown style

@Vvaradinov please familiarize yourself with our standard markdown style and always adopt it in your PRs from now on. E.g.

- add one empty line before and after every heading.
- no empty lines between the items of lists.

@Vvaradinov @MalteHerrmann Why is the markdown linter not failing on this?

Reply:
Please apply to the whole file and make the linter enforce a consistent style

Reply:
TBH I think an empty line between list items is completely fine since it makes PR reviews easier 🤷